docs/ruleset_functions/G311_exceptions/does_zone_meet_G3_1_1d.md

```diff
@@ -0,0 +1,67 @@
+# does_zone_meet_G3_1_1d  
+
+**Schema Version:** 0.0.28
+
+**Description:** determines whether a given zone meets the G3_1_1d exception "For laboratory spaces in a building having a total laboratory 
+exhaust rate greater than 15,000 cfm, use a single system of type 5 or 7 serving only those spaces.  The lab exhaust fan shall be modeled 
```

Comment:
this mentions building and not 'lab spaces'. so the exhaust air flow needs to be calculated for the entire building and if the building exhaust airflow >15,000 cfm, then we need to find spaces with space function = lab, then those zones need to use system 5/7

Reply:
the function has been updated to calculate building total lab exhaust